pages/glossary/decorative.md

```diff
@@ -1,6 +1,8 @@
 ---
 title: Decorative
 key: decorative
+unambiguous: false
```

Comment:
Can you explain why you think this is ambiguous? Keep in mind, ambiguous in our context means it can mean multiple things. I don't think that's true here. Sure something like "aesthetic" is debatable, but that word doesn't have two different meanings.

Reply:
On that topic, what do you think of #1095?